addons/pos_hr/static/src/app/utils/select_cashier_mixin.js

```diff
@@ -98,10 +100,18 @@ export function useCashierSelector({ exclusive, onScan } = { onScan: () => {}, e
             wrongPinNotification();
             return;
         }
-
-        if (pinMatchEmployees.length > 1 || list) {
+        if (list && !pin && pos.getCashier()) {
```

Comment:
```suggestion
        if (list && pos.getCashier()) {
```

I don't know if pin check is needed here, since you cannot access Pin popup when pos.getcashier is true.